### ISSUE TITLE
Object detection validation 

### ISSUE THREAD

**Issue by HerbertGourout** [AlexeyAB/darknet#3332]
Hello AlexeyAB,
I'm coming to you again because I think I'll need your help. As part of my internship, I set up an object detection algorithm with 4 classes using your version of Darknet and used Darknet19 as architecture. First, it must be mentioned that for the moment I’m the first person to work on objects detection in this company. The results obtained are very satisfactory and the company is very happy. But there is still some doubt among business leaders on the approach used.

The doubt lies in overfitting problems, k-fold cross-validation and the rate (%) representativity of the classes in the train and the test.
1.	 Concerning overfitting, why the use of k fold cross-validation is not important to deal with overfitting problems in yolo? In the YoloV2 article, the author talks about batch normalization. I would like to know if batch normalization would be sufficient to avoid overfitting?

2.	Concerning the rate (%) of class in the train and the test here is what I gave as answer:
-	 For training I need for each of my classes of objects at least 500-2000 observations in the train  sample. Moreover, all my object classes must be present in the train and test set.
 But all this is new to the data scientist of my company and he would like me to bring scientific papers that explain it.

3.	It is necessary to divide dataset to 3 (train, val, test), only in 2 cases:
-	(train ~5%, val ~80%, test ~15%) - if you want to train on small Train-set, then Fine-tune on Val-set with frozen most of layers, and then check mAP on Test-set
-	(train ~80%, val ~10%, test ~10%) - if you want to use double-blind checking, you send Train and Val sets to another person, who will train on Train-set and check mAP on Val-set, then you want to check whether he cheat you and you receive the model from this person and you check it on Test-set that he didn't have

This answer comes from you, and it sounds so new to them too. So they want me to bring scientific papers  who best explain that.

For the three concerns, I don't know if it's possible to have scientific papers from you or other people that explain that. That's what my company asks me.
